docs/README.md

```diff
@@ -13,17 +13,18 @@ If you plan to build a webapp to run on Cozy, you'll probably want to use a simp
 
 ## CSS Styleguide
 
-Check the [styleguide](https://cozy.github.io/cozy-ui/styleguide) to see all the variables, mixins, classes, utilities and how to use them.
+Check the [styleguide](https://cozy.github.io/cozy-ui/styleguide) to see all the variables, mixins, classes, utilities and how to use them with CSS classes only
```

Comment:
s/Check the [styleguide](https://cozy.github.io/cozy-ui/styleguide) to see all the variables, mixins, classes, utilities and how to use them with CSS classes only/Check the [styleguide](https://cozy.github.io/cozy-ui/styleguide) to see all variables, mixins, classes, utilities and how to use them with only CSS classes/